core/templates/domain/objects/objects-domain.constants.ts

```diff
@@ -66,5 +66,211 @@ export const ObjectsDomainConstants = {
     NON_INTEGER_ELEMENTS: 'I18N_INTERACTIONS_RATIO_NON_INTEGER_ELEMENTS',
     INCLUDES_ZERO: 'I18N_INTERACTIONS_RATIO_INCLUDES_ZERO',
     EMPTY_STRING: 'I18N_INTERACTIONS_RATIO_EMPTY_STRING',
-  }
+  },
+
+  UNITS_MAPPING: {
```

Comment:
Let's make this naming clearer, maybe add a comment too. Perhaps UNIT_TO_NORMALIZED_UNIT_MAPPING and add a comment to explain what this does and why we need it. (That will help devs who need to update it in future.)

Also presumably there are other constants specifying units; we should have a frontend test to ensure that (a) the set of keys here is exactly equal to the set of units, and that (b) every mapped-to value here is a valid unit.

Reply:
I guess no, as we handle currency units on our own and their aliases are defined in object-domain-constants. The validation for other units is done by mathjs. So, I created an array of valid units a user can enter and used it for testing.

Although there are some unit related constants defined in third_party.js, but they are for internal use only.